Jenkinsfile

```diff
@@ -61,18 +61,51 @@ pipeline {
             }
         }
 
-        stage('Cypress Test') {
+        stage('Fix user rights') {
```

Comment:
What's this?

Reply:
Moved the command in the next stage. Added also as always post action